src/signing.c

```diff
@@ -124,6 +135,15 @@ void aws_crt_signable_release(aws_crt_signable *signable) {
     aws_signable_destroy(signable);
 }
 
+int aws_crt_signing_result_apply_to_http_request(const aws_crt_signing_result *result, aws_crt_http_message *request) {
+    return aws_apply_signing_result_to_http_request(request->message, aws_crt_allocator(), result);
+}
+
+void aws_crt_signing_result_release(aws_crt_signing_result *result) {
+    aws_signing_result_clean_up(result);
+    aws_mem_release(aws_crt_allocator(), result);
+}
```

Comment:
this is not reference counting

Reply:
Signing results are not reference counted at all. They are only available via callback